src/config/urls.py

```diff
@@ -1,7 +1,9 @@
 from django.contrib import admin
-from django.urls import include, path
+from django.http import HttpResponse
+from django.urls import include, path, re_path
 
 urlpatterns = [
     path("api/v1/", include("safe_apps.urls", namespace="v1")),
     path("admin/", admin.site.urls),
+    re_path(r"^check/", lambda request: HttpResponse("Ok"), name="check"),
```

Comment:
Better use path instad of `re_path`, I should change it too :)

Reply:
Well, I use `re_path` everywhere, so I might leave it that way as it's not deprecated

Reply:
Yeah I started with `path` but was wondering if I should change it to `re_path` as you have 🤔

Reply:
Can `path` have issues with ending the endpoint with `/` or without it? I've tested `check/` and `check` and both worked fine with `path`

Reply:
There's an auto redirect, but stick with `check/`